build/spas.js

```diff
@@ -103,11 +101,6 @@ async function buildSPAs(options) {
     console.log("Wrote", path.join(outPath, path.basename(url)));
   }
 
-  if (contributorSpotlightRoot) {
-    buildContributorSpotlight(options);
-    buildCount++;
-  }
-
   // Basically, this builds one (for example) `search/index.html` for every
   // locale we intend to build.
   for (const root of [CONTENT_ROOT, CONTENT_TRANSLATED_ROOT]) {
```

Comment:
Can we just use `VALID_LOCALES` here instead of relying on `CONTENT_ROOT` and `CONTENT_TRANSLATED_ROOT` and `fs`?

```js
// at the top of the file
const { VALID_LOCALES } = require("../libs/constants");

// here
for (const locale of VALID_LOCALES.keys()) {
```

Reply:
Let's do this in a separate PR.